flutter_app/lib/drone_information.dart

```diff
@@ -0,0 +1,70 @@
+import 'dart:math';
+import 'package:flutter/material.dart';
+import 'package:imacs/mavlink_communication.dart';
+
+import 'data_field_widget.dart';
+
+/// Widget to arrange multiple data widgets in tabular form.
+///
+/// This widget  displays all the data fetched from
+/// Mission Planner MAVLink. It automatically sets the data in a two
+/// column layout.
+///
+class DroneInformation extends StatelessWidget {
+  /// @brief Constructs a DroneInformation widget.
+  ///
+  /// @param comm The communication channel (MAVLink) where to get
+  /// the data from
+  ///
+  const DroneInformation({
+    super.key,
+    required this.comm,
+  });
+
+  final MavlinkCommunication comm;
+
+  @override
+  Widget build(BuildContext context) {
+    return GridView.count(
+      /// describes how many items in one row.
+      crossAxisCount: 2,
+
+      /// describes how spaced out things are.
+      childAspectRatio: 4,
```

Comment:
4 seem really high for this value, try experimenting with values between 1 and 2